prow/oss/plugins.yaml

```diff
@@ -67,6 +67,11 @@ lgtm:
   - googleforgames
   - grpc-ecosystem
   review_acts_as_lgtm: true
+- repos:
+  - GoogleCloudPlatform/testgrid
+  - GoogleCloudPlatform/oss-test-infra
+  review_acts_as_lgtm: true
+  trusted_team_for_sticky_lgtm: googlers
```

Comment:
Using this large team will increase API token usage and potentially latency.  It should be fine for these two repos, but if the LGTM plugin becomes sluggish this is likely the culprit.